providers/togglz/README.md

```diff
@@ -0,0 +1,7 @@
+# Unofficial Togglz OpenFeature Provider for Java
+
+## Usage
+See [TogglzProviderTest.java](./src/test/java/dev/openfeature/contrib/providers/togglz/TogglzProviderTest.java)
```

Comment:
It can be addressed in a follow-up PR, but please add more information to the readme. Please include information such as install instructions, basic usage, limitations (only booleans are supported), etc.